### PR TITLE
My first code review

### PR DESCRIPTION
**Pull request by Chimoneg27**
### This is my first code review.

_**_In this review I did the following_**_

- Kindly pointed out the learners' errors
- Gave suggestions on how he/she can fix the errors
- I made use of inline comments, where needed
- I also used a kind tone to guide the learner on what changes are to be made

### CODE REVIEW

**Review by Chimoneg27 — COMMENTED** [review]
### Your project is good, but make these changes.

ItsJustATinyChangeSheaWhitneyGIF

- I kindly suggest making use of semantic html next time. It improves the readability of your code
- Knowing CSS positioning is very important. It will allow your html elements to be well placed on the webpage. Please make use of this feature.
- May you please install the required linters. They are very important as they allow you to easily detect stylistic errors in your code.

File: css/style.css [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<<<<<<< HEAD
 main {
     display: flex;
```
Comment (Olamarx) [inline css/style.css:2]: - [ ] This is an example of inline comment.

File: css/style.css [hunk 2]
```diff
@@ -1,4 +1,3 @@
...
     flex-direction: column;
```
Comment (Chimoneg27) [inline css/style.css:3]: Good use of flexbox.

**Review by Chimoneg27 — COMMENTED** [review]
### Please make the following changes

CatDrivingSeriousGIF

-Add either CSS-flexbox or CSS-grid for your layout. These methods ensure certain content is properly placed on your web-page
- Please remove the comment from your index.html file, It is not necessary.
- Please use responsive values in your CSS stylesheet, like percentages, rems or even ems, instead of pixels. This makes sure that your html elements stay responsive across multiple devices.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Chimoneg27, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- GitHub review feature ✔️

## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly use [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) to make at least one inline comment so that you will know how to use it.
- [ ] Kindly use polite comments. Your comment "CSS positioning is a must. Your web page must look well organized" could be better written as "It is important you know that CSS positioning is a necessary aspect and as a result, I will implore you to position your content with it."
- [ ] I would recommend you also give a reason for your comment so that the student will not be making changes in the dark. Your comment "May you please install the required linters. They are very important" should let the student know why it is important.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 3]
```diff
@@ -1,4 +1,3 @@
-<<<<<<< HEAD
 main {
```
Comment (Chimoneg27) [inline css/style.css:1]: May you kindly add CSS media queries. They allow for responsiveness across a wide range media devices.

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @Bleu-Yves-Sopoude      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing some of the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights 
- nicely set up `.gitignore`✔️  
- Good job with using GitHub features  ✔️  
  
# Required Changes  🌵 

## Critical note 🔴 :
- [ ] It is a great offense to delete comments by a code reviewer in your project. As I believe this is the first, and you're pardoned. Constant repetition of such will get you penalized by leaving the program. These comments are there to help you improve your code and make you a better programmer 👍

## Deleted comments:
image

- [ ] As you've deleted some comments, you're meant to have at least two. Kindly add one or more with the same directives as given in the project requirements 👍

## Requirements:
image
  

- [ ] Your pull request is of utmost importance. kindly make it very descriptive as you've done with other projects


_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @Chimoneg27, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

-  Make use of inline comment :heavy_check_mark:
- Give review as expected :heavy_check_mark:

### Changes Required

- [x] As requested by the reviewer, I would like you to implement the changes he has requested for, kindly add at least two inline comment as this is part of the project requirement. Please kindly implement this as it will help you improve your self better when given reviews. Also be sure to follow other suggestions raised by the first reviewer when given the review.
Thanks

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 3 above)
Comment (usorfaitheloho) [reply css/style.css:1]: You did a good job adding a polite review with an explanation as well, please kindly add another inline comment review as suggested by the previous reviewers 😄

File: css/style.css [hunk 4]
```diff
@@ -151,7 +150,8 @@ list-style: none;
```
Comment (Chimoneg27) [inline css/style.css:150]: -Please add some colors. This will brighten up your web page, making it more attractive.

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
Hi @Chimoneg27 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Use inline comments
- made a polite review

 ## Required Changes ♻️

- [ ] You did a good job adding a polite review with an explanation as well, please kindly add another inline comment review as suggested by the previous reviewers 😄 

- [ ] Please edit your PR to make it descriptive PR, add a title and summary explaing what you're working on. This was also suggested by the first reviewer

If you have any question you can message me on slack @faith eloho usor 👍 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Chimoneg27, 👋 👋  

Your project is complete! Great job for following the project requirement🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Make the changes requested by reviewers ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**